ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/router_isis.py

```diff
@@ -57,9 +56,9 @@ def router_isis(self) -> dict | None:
             }
         ti_lfa_protection = get(self._hostvars, "isis_ti_lfa.protection")
         if ti_lfa_protection == "link":
-            router_isis["isis_af_defaults"].append("fast-reroute ti-lfa mode link-protection")
+            router_isis["address_family_ipv4"].update({"fast_reroute_ti_lfa": {"mode": "link-protection"}})
```

Comment:
don't use update for only one key
```suggestion
            router_isis["address_family_ipv4"]["fast_reroute_ti_lfa"] = {"mode": "link-protection"}